src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/featuredvideo/FeaturedVideoComponentObject.java

```diff
@@ -0,0 +1,164 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.featuredvideo;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.Cookie;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class FeaturedVideoComponentObject extends WikiBasePageObject {
+
+  private static final String AUTOPLAY_COOKIE = "featuredVideoAutoplay";
+
+  @FindBy(css = ".featured-video")
+  private WebElement featuredVideo;
+
+  @FindBy(css = ".wikia-jw-settings-button")
+  private WebElement settingsMenu;
+
+  @FindBy(css = ".jw-title-primary")
+  private WebElement primaryTitle;
+
+  @FindBy(css = ".jw-title-secondary")
+  private WebElement secondaryTitle;
+
+  @FindBy(css = "#featured-video__player")
+  private WebElement player;
+
+  @FindBy(css = ".jw-controlbar")
+  private WebElement controlbar;
+
+  @FindBy(css = ".jw-svg-icon-play")
+  private WebElement controlbarPlayIcon;
+
+  @FindBy(css = ".jw-svg-icon-pause")
+  private WebElement controlbarPauseIcon;
+
+  @FindBy(css = ".video-feedback")
+  private WebElement videoFeedback;
+
+  @FindBy(css = "#featured-video__player-videoAutoplayToggle")
+  private WebElement autoplayToggle;
+
+  @FindBy(css = ".jw-icon-tooltip.jw-icon-volume.jw-off")
+  private WebElement volumeMuted;
+
+  @FindBy(css = ".wikia-jw-settings__quality-button")
+  private WebElement videoQualityButton;
+
+  @FindBy(css = ".wikia-jw-settings__captions-button")
+  private WebElement videoCaptionsButton;
+
+  @FindBy(css = ".wikia-jw-settings__submenu")
+  private List<WebElement> videoSettingsSubmenu;
+
+  public FeaturedVideoComponentObject() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1186)

Reply:
@qaga if constructor is empty, you can just remove it